modules/stanford_events_importer/tests/src/Kernel/Plugin/Field/FieldWidget/StanfordEventsImporterAPIURLFieldWidgetTest.php

```diff
@@ -18,42 +23,61 @@ class StanfordEventsImporterAPIURLFieldWidgetTest extends KernelTestBase {
   /**
    * {@inheritDoc}
    */
-  protected static $modules = [
+  public static $modules = [
     'system',
-    'path_alias',
     'node',
     'user',
     'link',
-    'datetime',
     'field',
-    'stanford_events_importer'
+    'stanford_events',
+    'stanford_events_importer',
   ];
 
+  // https://www.drupal.org/project/drupal/issues/3039565
+  protected $strictConfigSchema = FALSE;
+
   /**
    * {@inheritDoc}
    */
   protected function setUp() {
     parent::setUp();
     $this->installEntitySchema('user');
     $this->installEntitySchema('node');
-    $this->installEntitySchema('field_config');
+    $this->installConfig(['system', 'field', 'link']);
```

Comment:
I don't think its needed, but its probably only 2 or 3 files. :man_shrugging:

Reply:
It yelled at me for not having schema.

Reply:
this install config, not schema. they be different